inc/html.class.php

```diff
@@ -4222,9 +4222,16 @@ static function printPager($start, $numrows, $target, $parameters, $item_type_ou
             echo Html::hidden('item_type_param',
                               array('value' => Toolbox::prepareArrayForInput($item_type_output_param)));
          }
-         $split = explode("&amp;",$parameters);
 
-         for ($i=0 ; $i<count($split) ; $i++) {
+         $parameters = trim($parameters, '&amp;');
+         if (strstr($parameters, 'start') === false) {
+            $parameters .= "&amp;start=$start";
+         }
+
+         $split = explode("&amp;", $parameters);
+
+         $count_split = count($split);
+         for ($i=0 ; $i<$count_split ; $i++) {
```

Comment:
if we need cherrypick the commit, is this ok for cs check ? space before ';' and missing spaces around '<'